gradle.properties

```diff
@@ -34,3 +33,9 @@ httpMockServer=5.12.0
 micrometerVersion=1.8.2
 bouncycastleVersion=1.70
 gatlingVersion=3.7.5
+edcDeveloperId=mspiekermann
+edcDeveloperName=Markus Spiekermann
+edcDeveloperEmail=markus.spiekermann@isst.fraunhofer.de
```

Comment:
@mspiekermann are you cool with adding you as "the man"?

Reply:
It is possible to provide more than one developer